src/js/constants/ServiceSchema.js

```diff
@@ -45,7 +45,7 @@ let SERVICE_SCHEMA = {
           type: 'number',
           default: 1,
           getter: function (service) {
-            return (service.getInstancesCount() || 0);
+            return (service.getInstancesCount() || 0) + '';
```

Comment:
Well the `type` there is the `input type="number"` i think. Maybe the string conversion is needed because reading an input's value (`input.value`) would return a string anyway?